airbyte-webapp/src/packages/cloud/cloudRoutes.tsx

```diff
@@ -95,7 +98,7 @@ const MainRoutes: React.FC = () => {
         <Route path={`${RoutePaths.Settings}/*`} element={<CloudSettingsPage />} />
         <Route path={CloudRoutes.Credits} element={<CreditsPage />} />
 
-        {workspace.displaySetupWizard && !hideOnboardingExperiment && (
+        {(workspace.displaySetupWizard || isExperimentVariant) && !hideOnboardingExperiment && (
```

Comment:
Maybe to prevent this we could actually check the `hideOnboardingExperiment` in the banner and link to connections directly as part of this PR when the onboarding page doesn't exists? That way both experiments would always run.

Reply:
Absolutely. Will change it

Reply:
✅ Addressed so both experiments can live together